### PR TITLE
04/14 Upcoming Releases Update

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 1]
```diff
@@ -10,21 +10,33 @@ The teleport team delivers a new major release roughly every 3 months.
 
 ### Teleport 9.1
 
-Teleport 9.1 will add Database and Kubernetes Access support to Machine ID and
-server-side filtering and pagination to the Web UI.
+Teleport 9.1 will add preview support for Postgres/CockroachDB backends and
+server-side filtering and pagination for CLI and the Web UI.
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:14]:
```suggestion
server-side filtering and pagination for the CLI and Web UI.
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 2]
```diff
@@ -10,21 +10,33 @@ The teleport team delivers a new major release roughly every 3 months.
...
 | Version     | Date             |
 |-------------|------------------|
-| Release     | Early April 2022 |
+| Release     | April 18th 2022  |
 
 ### Teleport 9.2
 
-Teleport 9.2 will include Teleport Terminal, a native application that can be
-used to access Teleport Servers and Databases.
+Teleport 9.2 will add Database and Kubernetes Access support to Machine ID and
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:22]: Just to double-check, does this mean that we had clickable label filters already but we're adding Database and Kubernetes Access support to them?

File: docs/pages/preview/upcoming-releases.mdx [hunk 3]
```diff
@@ -10,21 +10,33 @@ The teleport team delivers a new major release roughly every 3 months.
...
+clickable label filters.
+
+In addition, Teleport 9.2 will include Teleport Terminal, a native application
+that can be used to access Teleport Servers and Databases.
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:26]:
```suggestion
that can be used to access Teleport Nodes and Databases.
```

**Review by ptgott — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 4]
```diff
@@ -10,21 +10,34 @@ The teleport team delivers a new major release roughly every 3 months.
...
 
-Teleport 9.1 will add Database and Kubernetes Access support to Machine ID and
-server-side filtering and pagination to the Web UI.
+Teleport 9.1 will add preview support for Postgres/CockroachDB backends and
+server-side filtering and pagination for the CLI and the Web UI.
 
 | Version     | Date             |
 |-------------|------------------|
-| Release     | Early April 2022 |
+| Release     | April 18th 2022  |
```
Comment (xinding33) [inline docs/pages/preview/upcoming-releases.mdx:18]: Will we have something ready for April 18th? I thought we were targeting April 29th now?

File: docs/pages/preview/upcoming-releases.mdx [hunk 5]
```diff
@@ -10,21 +10,34 @@ The teleport team delivers a new major release roughly every 3 months.
 
 ### Teleport 9.1
 
-Teleport 9.1 will add Database and Kubernetes Access support to Machine ID and
-server-side filtering and pagination to the Web UI.
+Teleport 9.1 will add preview support for Postgres/CockroachDB backends and
+server-side filtering and pagination for the CLI and the Web UI.
```
Comment (xinding33) [inline docs/pages/preview/upcoming-releases.mdx:14]:
```suggestion
Teleport 9.1 will add:
- Teleport Terminal preview (pending)
- Machine ID X.509 certificate support (Database and Kubernetes Access) & TLS routing
- Postgres and CockroachDB backends preview
- Server-side filtering and pagination for the CLI and Web UI
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 6]
```diff
@@ -10,21 +10,34 @@ The teleport team delivers a new major release roughly every 3 months.
...
-used to access Teleport Servers and Databases.
+Teleport 9.2 will add Database and Kubernetes Access support for Machine ID and
+clickable label filters for the Web UI.
+
+In addition, Teleport 9.2 will include Teleport Terminal, a native application
+that can be used to access Teleport Nodes and Databases.
+
+| Version     | Date            |
+|-------------|-----------------|
+| Release     | Early May 2022  |
```
Comment (xinding33) [inline docs/pages/preview/upcoming-releases.mdx]: I thought we were planning on shipping Machine ID X.509 support and Teleport Terminal preview in 9.1 (and delaying that release to 4/29).

**Pull request merged**